client/my-sites/domains/domain-management/components/breadcrumbs/index.jsx

```diff
@@ -34,10 +34,7 @@ const Breadcrumbs = ( { items, mobileItem, buttons, mobileButtons, className } )
 	const renderItemLabel = ( item ) => {
 		if ( item.href ) {
 			return (
-				<a
-					className="breadcrumbs__item-label breadcrumbs__item-label--clickable"
-					href={ item.href }
-				>
+				<a className="breadcrumbs__item-label is-clickable" href={ item.href }>
```

Comment:
Is there any reason for the `is-clickable` class? I think it can be removed:

```suggestion
				<a className="breadcrumbs__item-label" href={ item.href }>
```
This should be good to go, as I couldn't find any styles defined for it specifically. 
Neither for the `breadcrumbs__item-label--clickable` class. 
I wonder why I added it in the first place 🤔

Reply:
That's true, it isn't used, good point 🤔 ok, I'll remove it, if we need we can readd it in the future 🙂